openedx/core/djangoapps/credit/services.py

```diff
@@ -159,3 +159,52 @@ def set_credit_requirement_status(self, user_id, course_key_or_id, req_namespace
             status,
             reason
         )
+
+    def remove_credit_requirement_status(self, user_id, course_key_or_id, req_namespace, req_name):
+        """
+        A simple wrapper around the method of the same name in api.eligibility.py. The only difference is
```

Comment:
nit: Please make sure that docstrings do not exceed the 80 characters limit: https://github.com/edx/edx-platform/wiki/Python-Guidelines#follow-pep-8